Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -54,6 +54,18 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         NotificationCenter.default.post(name: notificationIdentifier, object: nil)
         XCTAssertFalse(viewController.notificationFired)
     }
+    
+    class MyViewController: UIViewController {
+        var foo: String?
+    }
```

Comment:
Move this declaration to the top of the class, under `MockNotificationViewController`.